README.md

```diff
@@ -20,6 +20,7 @@ it on [Google Play](https://play.google.com/store/apps/details?id=org.wordpress.
 Notes:
 
 * To use WordPress.com features (login to WordPress.com, access Reader and Stats, etc) you need a WordPress.com OAuth2 ID and secret. Please read the [OAuth2 Authentication](#oauth2-authentication) section.
+* While building the app in Android Studio ignore all the update plugin prompts (updating plugins may resulted in plugin conflicts and interrupt the build process).
```

Comment:
Thanks for the PR @ajithrn !

What do you think about narrowing the scope of this hint? Do you perhaps mean some particular plugin to avoid updating? Currently the hint is a bit too general.

Reply:
@hypest avoid updating Gradle since this project depends on 5.4.1 and all the version above 6.0 wont support gradle.build-scan hence the app wont compile.

Reply:
Aha, I see. Yes, we try to avoid that update if the effort to make it work is not planned. Good find!

How about we put something like this to the readme then:
```
* While loading/building the app in Android Studio ignore the prompt to update the gradle plugin version as that will probably introduce build errors. On the other hand, feel free to update if you are planning to work on ensuring the compatibility of the newer version.
```

Reply:
sounds great.